services/metadata/backend/data/db.json

```diff
@@ -5720,6 +5720,2467 @@
                     ]
```

Comment:
since we're nearing to go online with this... should we remove all test projects, and only leave in the real projects and Rosetta? Or is that too early?

Reply:
IMO it's still time to do it. I'll create a task for the final preparation and will add this as a one of that task points.

Reply:
perfect